adal/src/telemetry/java/com/microsoft/aad/adal/EventStrings.java

```diff
@@ -165,6 +165,14 @@
 
     static final String BROKER_ACCOUNT_SERVICE_CONNECTED = EVENT_PREFIX + "broker_account_service_connected";
 
+    static final String BROKER_ACCOUNT_SERVICE_CONNECTION_ERROR_INFO = EVENT_PREFIX + "broker_account_service_connection_error_info";
+
+    static final String BROKER_ACCOUNT_SERVICE_ERROR = EVENT_PREFIX + "broker_account_service_error";
```

Comment:
>ERVICE_ERROR [](start = 40, length = 12)

this is the subcode and above one is the trace?

Reply:
Yep, that's correct 👍